src/Input/Select/Select.tsx

```diff
@@ -197,6 +197,14 @@ class Select extends React.Component<Props, State> {
     } else if (e.keyCode === 13) {
       inputElement.blur();
       const activeElement = this.getActiveElement();
+      const activeElementIndex = Number(get(activeElement, 'dataset.id'));
```

Comment:
Since `Number(null) === 0`, `parseInt` might be safer. But might need `isNaN` check as well.

Reply:
Oh maybe i'll do `Number(get(activeElement, 'dataset.id', undefined));` instead. Will that work?

Reply:
There also shouldn't be a way for `dataset.id` to be `undefined`. 🤔 
I think while `dataset.id` is `undefined`, `activeElement` should be `undefined` as well.
Therefore, we might need the condition `if(!activeElement) return;` after line 199.